skiko/build.gradle.kts

```diff
@@ -806,6 +806,7 @@ fun createCompileJvmBindingsTask(
         }
         OS.Linux -> {
             includeHeadersNonRecursive(jdkHome.resolve("include/linux"))
+            includeHeadersNonRecursive(runPkgConfig("dbus-1"))
```

Comment:
Not sure if it is required, and forces pkg-config dependency. Could it be removed?

Reply:
There is any another utility that I could use the replace it? I did not found any other solution to resolve libdbus headers.

Reply:
OK, as CI seems to work let's keep it.